procedures/statement_performance_analyzer.sql

```diff
@@ -0,0 +1,792 @@
+                SET @sys.tmp.SQL = CONCAT(@sys.tmp.SQL, ' LIMIT ', @sys.statement_analyzer_limit);
+            END IF;
+            IF (@sys.debug = 'ON') THEN
+                SELECT @sys.tmp.SQL AS 'Debug';
+            END IF;
+            PREPARE stmt_select_view FROM @sys.tmp.SQL;
+            EXECUTE stmt_select_view;
+            DEALLOCATE PREPARE stmt_select_view;
+        END IF;
+
+        IF (FIND_IN_SET('with_sorting', in_views)) THEN
+            SELECT CONCAT('Top ', @sys.statement_analyzer_limit, ' Queries with Sorting') AS 'Next Output';
+            SET @sys.tmp.SQL = REPLACE(
+                                  (SELECT VIEW_DEFINITION
+                                     FROM information_schema.VIEWS
+                                    WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'statements_with_sorting'
+                                  ),
+                                  '`performance_schema`.`events_statements_summary_by_digest`',
+                                  v_digests_table
+                               );
+            IF (@sys.statement_analyzer_limit > 0) THEN
+                SET @sys.tmp.SQL = CONCAT(@sys.tmp.SQL, ' LIMIT ', @sys.statement_analyzer_limit);
+            END IF;
+            IF (@sys.debug = 'ON') THEN
+                SELECT @sys.tmp.SQL AS 'Debug';
+            END IF;
+            PREPARE stmt_select_view FROM @sys.tmp.SQL;
+            EXECUTE stmt_select_view;
+            DEALLOCATE PREPARE stmt_select_view;
+        END IF;
+
+        IF (FIND_IN_SET('with_temp_tables', in_views)) THEN
+            SELECT CONCAT('Top ', @sys.statement_analyzer_limit, ' Queries with Internal Temporary Tables') AS 'Next Output';
+            SET @sys.tmp.SQL = REPLACE(
+                                  (SELECT VIEW_DEFINITION
+                                     FROM information_schema.VIEWS
+                                    WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'statements_with_temp_tables'
+                                  ),
+                                  '`performance_schema`.`events_statements_summary_by_digest`',
+                                  v_digests_table
+                               );
+            IF (@sys.statement_analyzer_limit > 0) THEN
+                SET @sys.tmp.SQL = CONCAT(@sys.tmp.SQL, ' LIMIT ', @sys.statement_analyzer_limit);
+            END IF;
+            IF (@sys.debug = 'ON') THEN
+                SELECT @sys.tmp.SQL AS 'Debug';
+            END IF;
+            PREPARE stmt_select_view FROM @sys.tmp.SQL;
+            EXECUTE stmt_select_view;
+            DEALLOCATE PREPARE stmt_select_view;
+        END IF;
+
+        IF (FIND_IN_SET('custom', in_views)) THEN
+            SELECT CONCAT('Top ', @sys.statement_analyzer_limit, ' Queries Using Custom View') AS 'Next Output';
+
+            IF (@sys.statement_analyzer_view IS NULL) THEN
+                SET @sys.statement_analyzer_view = sys.sys_get_config('sys.statement_analyzer_view', NULL);
```

Comment:
Same here with consistency in naming. And lack of sys_config row for both of the variables too.